src/Sulu/Bundle/AdminBundle/Resources/js/components/Table/Table.js

```diff
@@ -19,11 +19,23 @@ type Props = {
     buttons?: Array<ButtonConfig>,
     /** Can be set to "single" or "multiple". Defaults is "none". */
     selectMode?: SelectMode,
+    /** Can be set to  */
```

Comment:
This comment is not finished, and I wouldn't add a comment here as well, as it doesn't add any value.